README.md

```diff
@@ -219,3 +219,10 @@ other frameworks could adopt Puppeteer as their foundational layer.
 The past few months have brought [several new libraries for automating headless Chrome](https://medium.com/@kensoh/chromeless-chrominator-chromy-navalia-lambdium-ghostjs-autogcd-ef34bcd26907). As the team authoring the underlying DevTools Protocol, we're excited to witness and support this flourishing ecosystem.
 
 We've reached out to a number of these projects to see if there are opportunities for collaboration, and we're happy to do what we can to help.
+
+#### Q: What features does Puppeteer not support?
+
+You may find that Puppeteer does not behave as expected when controlling pages that incorporate audio and video. (For example, [video playback/screenshots is likely to fail](https://github.com/GoogleChrome/puppeteer/issues/291).) There are two reasons for this:
+
+* Puppeteer is bundled with Chromium--not Chrome--and so by default, it inherits all of [Chromium's video-related limitations](https://www.chromium.org/audio-video). This means that Puppeteer does not understand the MP4 container format, and also does not support MP3, AAC or H.264. (However, it is possible to force Puppeteer to use a separately-installed version Chrome instead of Chromium via the [`executablePath` option to `puppeteer.launch`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions). You should only use this configuration if you need Chrome's audio/video support.)
```

Comment:
Thanks for adding this. mp3/mp4 support was added to chromiume as of 2 days ago. https://bugs.chromium.org/p/chromium/issues/detail?id=746579

We should probably just leave those up next time we rev chromium.

Reply:
Sure, that's good news! Updated to remove mention of MP3/MP4.